Header header
string child_frame_id
geometry_msgs/Pose[] poses
geometry_msgs/Twist[] twist